src/package.json

```diff
@@ -14,8 +14,7 @@
     "express": "~4.16.0",
     "express-cluster": "0.0.5",
     "http-errors": "~1.6.2",
-    "jade": "~1.11.0",
-    "kafka-node": "~2.6.1",
+    "kafka-node": "^5.0.0",
```

Comment:
are the any changes in the syntax or breaking changes with major version update ?